com.wireframesketcher.WireframeSketcher.json

```diff
@@ -0,0 +1,81 @@
+{
+  "id": "com.wireframesketcher.WireframeSketcher",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [],
+  "command": "WireframeSketcher",
+  "separate-locales": false,
+  "tags": [
+    "proprietary"
+  ],
+  "rename-appdata-file": "wireframesketcherstudio.appdata.xml",
+  "rename-desktop-file": "wireframesketcherstudio.desktop",
+  "rename-icon": "WireframeSketcher",
+  "finish-args": [
+    "--require-version=1.7.1",
+    "--filesystem=host",
```

Comment:
why do yo need host filesystem access?

Reply:
This application is based on Eclipse and basically has the same requirements. Eclipse is distributed via flathub as  org.eclipse.Java app.

Reply:
Removed 1.7.1 version requirement.

Reply:
Also switched to home requirement. WireframeSketcher uses a workspace that needs to be placed in user's home.